index.js

```diff
@@ -39,10 +50,4 @@ class ImmediateStore {
   }
 }
 
-function nextTick (cb, err, val) {
-  process.nextTick(() => {
-    if (cb) cb(err, val)
```

Comment:
Previously, the function worked even when `cb` was not passed in. The updated code doesn't work when `cb` is not a function.